src/domUpdates.js

```diff
@@ -218,8 +219,24 @@ const closeRecipe = () => {
   clickedRecipe.classList.add("hidden");
   clickedRecipe.classList.remove("flex");
   clickedRecipe.classList.remove("fade-in");
+  ingredientsList.innerHTML = '';
 };
 
+const getIngredients = currentRecipeCard => {
+  createIngredientsHTML(currentRecipeCard.ingredients);
+};
+
+const createIngredientsHTML = ingredients => {
+  ingredients.forEach((ingredient, i) => {
+    let ingredientLabelName = `ingredient${i}`
+    ingredientsList.innerHTML += `
+    <label for=${ingredientLabelName}>
+      <input id=${ingredientLabelName} type="checkbox" name="${ingredientLabelName}" />
+      ${ingredient.amount} ${ingredient.unit} ${ingredient.name}
+    </label>
```

Comment:
Look good! I think we may just technically be missing the quotes for `for` and `id` but other than that looks good to me

Reply:
done!